### ISSUE TITLE
Add Screen Reader Only Text to the Insert Link menu for Accessibility WCAG 2.4.4 (Link Purpose)

### ISSUE THREAD

**Issue by padarr** [tinymce/tinymce#11253]
## 📝 Provide a description of the new feature or improvement
When using the Wave Tool I encountered the problem of a Phone number link failing because it didn't have any instructional text associated with it. Tiny MCE is included in the CMS Terminal Four and as an Admin I have access to the option for "Source Code" and can easily add the needed text. However, the problem that occurs is that a contributor that should not have access at the code level, doesn't have a way to enter the extra sr-only text.  Adding a Screen Reader Only Text field with the option to make it pre or post or possibly both a pre/post option in the "Insert Link" tool menu can solves this providing a contributor the ability to include instructional text for use by screen readers.
`<p>Phone <a href="tel:+18005550199"><span class="sr-only">Call </span>800-555-0199</a>.</p>`
Without the sr-only span, when a user is tabbing to the tel link it only reads the number and not the surrounding text so it gives an alert in the WAVE tool to provide instruction text.

### 🫶 What is the motivation?
Without the sr-only span, when a user is tabbing to the tel link it only reads the number and not the surrounding text so it gives an alert in the WAVE tool to provide instruction text.

### 🔗 What is the consequence of not having this feature?
The WAVE tool flags this because it is checking for WCAG 2.4.4 (Link Purpose). WAVE flags raw phone numbers as an "Alert" or structural note because a automated scanner cannot guarantee that a screen reader user will know what the number is for or who it connects to when tabbing through a list of links.

### 🚦 How important would you rate the requested feature or improvement?
An argument can be made for all three categories nice-to-have, important or critical. 
Nice to have: Opens it up to contributors being able to maintain in a CMS usage case.
Important: Helps with accessibility requirements.
Critical: It's an Accessibility Compliance issue, however there is a technical way to do this, the people maintaining content in the CMS cannot regularly do this without source code level access.  

---

If you'd like to see this implemented sooner, add a 👍 reaction to this post.
